src/pages/signUp/signupPageForm.jsx

```diff
@@ -0,0 +1,30 @@
+import { useRef } from 'react';
+import { useHistory } from 'react-router-dom';
+import classes from './signupPageForm.module.css';
+
+const SignupPageForm = () => {
+    const history = useHistory();
+    const userNameInputRef = useRef();
+
+    const submitHandler = (event) => {
+        event.preventDefault();
```

Comment:
What is this for?

Reply:
`event.preventDefault()`: The main function, the method tells the user agent that if the event does not get explicitly handled, its default action should not be taken as it normally would be. The event continues to propagate as usual.